src/test/pmem2_memcpy/TESTS.py

```diff
@@ -14,7 +14,8 @@
 class Pmem2Memcpy(t.Test):
     test_type = t.Short
     filesize = 4 * t.MiB
-    envs = ()
+    envs0 = ()
```

Comment:
Can the test changes be done with the `add_params` API? You'd have two params

Reply:
Yes, it's possible to do something similar to what I did in 1737d32989ebabe4ef38b6a10b69f47183b351ba. But the code would have to be shared between tests and that requires significant refactoring.